Resources/Prototypes/_RMC14/Traits/cm_neutral.yml

```
@@ -51,3 +51,11 @@
   description: trait-southern-desc
   components:
   - type: SouthernAccent
+
+- type: trait
+  id: ScottishAccent
+  name: trait-scottish-accent-name
+  description: trait-scottish-accent-desc
+  components:
+    - type: ReplacementAccent
+      accent: dwarf
```

By convention we put sequence nodes (the dash) on the same level as the node before (components)
```suggestion
  - type: ReplacementAccent
    accent: dwarf
```